cmake/projects/drm/hunter.cmake

```diff
@@ -22,7 +23,17 @@ hunter_add_version(
 )
 
 hunter_configuration_types(drm CONFIGURATION_TYPES Release)
-hunter_pick_scheme(DEFAULT drm)
+hunter_pick_scheme(DEFAULT url_sha1_autotools)
+list(APPEND drm_dependencies
```

Comment:
This file can be processed several times so it should be `set(drm_dependencies ...)`. Also use names like `_drm_dependencies` since such module included into user's space.